conf/application.conf

```diff
@@ -162,6 +162,13 @@ microservice {
       host = localhost
       port = 8500
     }
+
+    destination {
```

Comment:
Could we maybe rename this to transit-movements-trader-at-destination? Just so it's clear where it's going (_destination_ could be generic)

Reply:
Chris can be done as we followed the same naming convention as frontend, we can discuss